src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -1500,7 +1500,7 @@ public class DateToPackageLabelConverter : IValueConverter
     {
         public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
         {
-            if (!(value is PackageManagerSearchElement packageManagerSearchElement)) return Visibility.Collapsed;
+            if (!(value is Dynamo.PackageManager.PackageManagerSearchElement packageManagerSearchElement)) return String.Empty;
```

Comment:
can you write a test for this converter?

Reply:
Will do, Mike!

Reply:
I think I covered all the test cases with the Test